docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs-material
+mkdocs-monorepo-plugin
```

Comment:
Not a change request, but from the `mkdocs-monorepo-plugin` [repo](https://github.com/backstage/mkdocs-monorepo-plugin/tree/master), it looks like this plugin is very helpful for keeping documentation closer to the code/relevant packages, and also supports merging documentation across multiple repositories. That will be very useful, nice find.

Reply:
Yep, it's pretty cool. One of the significant limitations is that it doesn't support nested monorepos.